### ISSUE TITLE
Font Weight on Header Buttons

### ISSUE THREAD

**Issue by changecourse** [mozilla-lockbox/lockbox-ios#272]
Use SF Regular for Action buttons instead of Semibold/Bold. 

For any text buttons in the header bar, let's use a regular font weight instead of a bolder one...

Back buttons (for settings and entry details) appear to be correct, but 'Cancel' and 'Done' appear to be bolder.

**Comment by devinreams** [comment]
I found one case where a "Done" button is seen under Settings > "Send Feedback" or "FAQ".

https://github.com/mozilla-lockbox/lockbox-ios/blob/4fc6eeff714dea9d5ae8be0383f61b3cc1075ef9/lockbox-ios/View/SettingListView.swift#L118-L119

The button was intentionally set to `.done` style: https://developer.apple.com/documentation/uikit/uibarbuttonitemstyle

>The style for a done button—for example, a button that completes some task and returns to the previous view.

A good argument to keep this as-is would be if Apple changed the done buttons to be different. Our app might then not be consistent with buttons meant to 'close'. 



That said, given what I found below, I think we can just change this to a `plain` style instead for consistency...

---

The other case I found is the "Cancel" button like when a web view (to sign in to FxA) pops over which is already set to `plain` but we also have added "semibold"

https://github.com/mozilla-lockbox/lockbox-ios/blob/4004c96681457adf12557b077d4d85b61cdc8341/lockbox-ios/View/FxAView.swift#L44-L47

https://github.com/mozilla-lockbox/lockbox-ios/blob/4004c96681457adf12557b077d4d85b61cdc8341/lockbox-ios/View/FxAView.swift#L54

So even though it appears bold, and may be inconsistent, it seems like an iOS-level design decision that we should remove the bold and keep as "plain".

---

To recap:

- Settings view with "Done": change to `plain`
- FxA popover with "Cancel": keep as `plain` and remove extra "semibold" treatment

...then we get what iOS is providing. 

@mozilla-lockbox/ux I think this captures your intention here...
